api/src/main/java/org/pmiops/workbench/cohortbuilder/FieldSetQueryBuilder.java

```diff
@@ -501,4 +514,29 @@ public QueryConfiguration buildQuery(ParticipantCriteria participantCriteria,
     }
     return results;
   }
+
+  public Iterable<Map<String, Object>> materializeTableQuery(TableQueryAndConfig tableQueryAndConfig,
```

Comment:
Should document what the output contains. Also, is it possible to type the value more tightly than "Object"?